radiorec.py

```diff
@@ -123,15 +125,30 @@ def record(args):
     if streamurl.endswith('.m3u'):
         verboseprint('Seems to be an M3U playlist. Trying to parse...')
         pool = urllib3.PoolManager()
-        with pool.request('GET',streamurl) as remotefile:
-            for line in remotefile:
-                if not line.decode('utf-8').startswith('#') and len(line) > 1:
-                    tmpstr = line.decode('utf-8')
+        try:
+            remotefile = pool.request('GET', streamurl)
+        except MaxRetryError:
+            print('The URL of the station is not found! Check' + args.station + ' in the Settings!')
```

Comment:
Errors should be indicated to stderr, not stdout. With this you're able to filter out the debug messages or info messages if you just would like to extract the errors (e.g. for sending automatically reports via scripts).
So i recommend to get a log object (via `logging.getLogger(__name__)`) and log it with `error` function.
E.g.:
`logging.getLogger(__name__).error('The URL of the station is somehow faulty! Check' + args.station + ' in the Settings!')`

Would do this for this line as well as line 134.

Reply:
+line 142 + line 150